server/game/core/gameSteps/prompts/ResourcePrompt.ts

```diff
@@ -92,7 +92,7 @@ export class ResourcePrompt extends AllPlayerPrompt {
     protected resourceSelectedCards(player: Player) {
         if (this.selectedCards[player.name].length > 0) {
             for (const card of this.selectedCards[player.name]) {
-                player.resourceCard(card);
+                player.resourceCard(card, false);
```

Comment:
Might be better to just ping Chebato to add this to his PR so there isn't a merge collision. Also that way he can add tests for it.